web/src/components/storage/ProposalSpacePolicyField.jsx

```diff
@@ -260,6 +237,18 @@ const DeviceRow = ({
   onCollapse = noop,
   onChange = noop
 }) => {
+  // Generates the action value according to the policy.
+  const action = () => {
```

Comment:
I'm not sure if this (together with the `useEffect` below) is the best (ie. most usable) behavior, but I find it reasonable enough to get this merged. We can re-evaluate in the future based on user's feedback.